taichi/ir/frontend_ir.h

```diff
@@ -504,6 +504,26 @@ class GlobalVariableExpression : public Expression {
   TI_DEFINE_ACCEPT_FOR_EXPRESSION
 };
 
+class MatrixExpression : public Expression {
```

Comment:
Some comments to plain what this `Expression` represents. 

Since Frontend IR is fairly close to Python codes, it would be nice if we can further introduce what python code gets lowered to this Expression.